target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "yuncore,cpe830", "qca,qca9531";
+	model = "YunCore/KuWfi CPE830(D)";
+
+	chosen {
+		bootargs = "console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd";
```

Comment:
Why is noinitrd needed here and why do you set the console if you haven't got a UART port? 

> 0.000000] Kernel command line: console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd rootfstype=squashfs,jffs2

As you can see the kernel command line already has rootfstype=squashfs,jffs2 here. Thus I guess you can remove the whole "chosen" section here.

Please start over with a device where you haven't modified the U-Boot environment variables, because changes except for the bootcmd (of which my PR takes care of) should not be needed.

Reply:
@CodeFetch You may complain thousands of times

THe realisty is the same

Reply:
@grosjo Why do you set the console if you haven't got an UART port?

Reply:
Because, if you dare READING the comments here, then you will notcie that soem DO HAVE an UART, and some DO NOT

Honestly, stop your harassment

Reply:
> 0.000000] Kernel command line: console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd rootfstype=squashfs,jffs2

Do you see that you have "rootfstype=squashfs,jffs2" here twice? This is because the rootfstype is being autodetected. So no need for defining it here.

Reply:
> Because, if you dare READING the comments here, then you will notcie that soem DO HAVE an UART, and some DO NOT
> 
> Honestly, stop your harassment

Which comment? Pepe2k made clear CPE830D (hasn't got UART) is not a CPE830 (has got UART)